### PR TITLE
some more looks like support based on some findings in tileset testing

### PR DESCRIPTION
**Pull request by I-am-Erk**
#### Summary
```SUMMARY: Infrastructure "some more looks like support based on some findings in tileset testing"```

#### Purpose of change
Adds more looks like support, somewhat at random, to various items that appear somewhat infrequently and for which the fallback art is no more informative than a general purpose sprite.

#### Testing
I cannot test at this time. I will try to get to it tonight unless someone else does the honours. Very few possible ways this could cause bugs.

### CODE REVIEW

File: data/json/furniture_and_terrain/furniture-storage.json [hunk 1]
```diff
@@ -419,6 +422,7 @@
     "type": "furniture",
     "id": "f_clothing_rail",
     "name": "clothing rail",
+    "id": "f_rack",
```
Comment (esotericist) [inline data/json/furniture_and_terrain/furniture-storage.json]:
```suggestion
    "looks_like": "f_rack",
```

**Comment by esotericist** [comment]
Did we still want to use statue as a looks-like for mannequins?

**Comment by esotericist** [comment]
Aside from that error, btw, this appears to work well enough. It definitely reduced the number of symbols in my test span

**Comment by KorGgenT** [comment]
image
a couple items don't actually have their first tile, but that's just notes for the tileset. these are all the tiles from top to bottom, left to right. the one on the far top left is what the one next to it looks_like, but i decided i didn't want to do that for all of them.

**Pull request merged**